R/utils.R

```diff
@@ -205,3 +205,63 @@ argument_parser <- function(..., description=NULL)
 
     return(parse_arguments)
 }
+
+
+# TO DO: when processing multiple traits, exclude any rows 
+# that have NAs for all traits
+
+#' Save a Plink-formatted file listing all samples phenotyped
+#' for one or all traits in a dataset.
+#'
+#' @export
+#'
+#' @param phenotype_data (string)
+#'      File path to a csv file containing one named column of 
+#'      sample IDs and one or more named columns with phenotype
+#'      data.
+#' 
+#' @param id_column (string)
+#'      Text string with the header for the column containing
+#'      sample IDs.
+#' 
+#' @param trait_column (string)
+#'      (default NULL) Text string with the header for the desired
+#'      phenotype, if only processing one. If NULL, all non-ID
+#'      columns in the file will be processed. 
+#' @param output_dir (string)
+#'      The directory in which the Plink-formatted IDs file will be 
+#'      saved.
+#'
+#' @return A list containing (1) the file path to the Plink-formatted 
+#'      IDs file and (2) a vector of phenotyped IDs.
+#
+get_phenotyped_ids <- function(phenotype_data, id_column, trait_column = NULL, output_dir) {
```

Comment:
I think that named arguments with default values go after all required arguments.  Consider
`
get_phenotyped_ids <- function(phenotype_data, id_column, output_dir, trait_column = NULL) {`

Reply:
addressed all comments in [d2cc608](https://github.com/Palmer-Lab-UCSD/rattaca/pull/7/commits/d2cc608948b7bad2f5fa8c1bb2b5498b4d1a8e0e)

Reply:
Looks good!